site/src/main/.vuepress/config.js

```diff
@@ -1067,7 +1067,8 @@ var config = {
 							['DataQuery/LastQuery','最近时间戳 Last 查询'],
 							['DataQuery/AlignByDeviceQuery','按设备对齐查询'],
 							['DataQuery/FillFunction','空值填充'],
-		                    ['DataQuery/GroupByFillQuery', '降采样补空值查询']
+		          ['DataQuery/GroupByFillQuery', '降采样补空值查询'],
+							['DataQuery/OrderByTimeQuery', '按时间倒序查询']
```

Comment:
Code alignment is better :D
```suggestion
		                                        ['DataQuery/GroupByFillQuery', '降采样补空值查询'],
							['DataQuery/OrderByTimeQuery', '按时间倒序查询']
```